Armadillo.Agent/appsettings.json

```diff
@@ -7,6 +7,8 @@
       }
     },
 
-    "SubcaseDataProvider": "Report"
+  "SubcaseDataProvider": "Random",
```

Comment:
Agent readme says that Report data provider is used by default https://github.com/skalitin/Armadillo/blob/master/Armadillo.Agent/readme.md